app/src/main/java/com/google/android/sambadocumentsprovider/provider/SambaDocumentsProvider.java

```diff
@@ -230,6 +235,17 @@ public Cursor queryRoots(String[] projection) throws FileNotFoundException {
   }
 
   @Override
+  public void ejectRoot(String rootId) {
+    if(BuildConfig.DEBUG) Log.d(TAG, "Ejecting root: " + rootId);
```

Comment:
nit: Leave a space between `if` and the parenthesis.

This is something you can catch by self review.